packages/components/menu/src/MenuItem/MenuItem.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+export const getMenuItemStyles = () =>
+  css({
+    display: 'block',
+    width: '100%',
+    background: 'none',
+    border: 0,
+    outline: 'none',
+    textDecoration: 'none',
+    fontSize: tokens.fontSizeM,
+    lineHeight: tokens.lineHeightCondensed,
```

Comment:
```suggestion
    lineHeight: tokens.lineHeightM,
```

But based on the figma file, it looks like the height of each item is `32px`

I'm not sure if you want to achieve it with line height or other css attribute